### PR TITLE
chore: Add translation to enums

### PR DESCRIPTION
**Pull request by edgonzalez24**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Feature :sparkles:
- [ ] Code style update (formatting, renaming) :art:
- [ ] Refactoring (no functional changes, no api changes) :recycle:
- [x] Documentation content changes :memo:
- [ ] Other :zap:

## What is the new behavior?
- Update locale
- Apply translation to: 

```
api/index.md
api/enums/pinia.MutationType.md
```

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: api/enums/pinia.MutationType.md [hunk 1]
```diff
@@ -4,21 +4,21 @@ editLinks: false
 sidebarDepth: 3
 ---
 
-[API Documentation](../index.md) / [pinia](../modules/pinia.md) / MutationType
+[Documentación API](../index.md) / [pinia](../modules/pinia.md) / MutationType
```
Comment (raulwwq0) [inline api/enums/pinia.MutationType.md:7]: I believe that translate it as "Documentación de la API" sounds more natural. What do you think??

File: api/index.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-API Documentation
+Documentación API
 
-# API Documentation
+# Documentación API {#api-documentation}
 
 ## Modules
```
Comment (raulwwq0) [inline api/index.md:5]: This has to be translate as "Módulos"

File: api/enums/pinia.MutationType.md (hunk 1 above)
Comment (edgonzalez24) [reply api/enums/pinia.MutationType.md:7]: You're right! Change is ready

File: api/index.md (hunk 2 above)
Comment (edgonzalez24) [reply api/index.md]: Done

**Pull request merged**